packages/psmisc.rb

```diff
@@ -3,33 +3,38 @@
 class Psmisc < Package
   description 'PSmisc is a set of some small useful utilities that use the proc filesystem.'
   homepage 'https://gitlab.com/psmisc/psmisc'
-  @_ver = '23.3'
+  @_ver = '23.6'
   version @_ver
   license 'GPL-2'
   compatibility 'all'
   source_url "https://gitlab.com/psmisc/psmisc/-/archive/v#{@_ver}/psmisc-v#{@_ver}.tar.bz2"
-  source_sha256 'fe530b0a29902f8660481248fc19f6994927282b4fe0cd992121016144b95fa6'
+  source_sha256 '91573ca0a1a50bd491b7c3cbe400126b0dadef9a2e328030d6469bb2448e0794'
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/psmisc/23.3_armv7l/psmisc-23.3-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/psmisc/23.3_armv7l/psmisc-23.3-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/psmisc/23.3_i686/psmisc-23.3-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/psmisc/23.3_x86_64/psmisc-23.3-chromeos-x86_64.tar.xz'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/psmisc/23.6_armv7l/psmisc-23.6-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/psmisc/23.6_armv7l/psmisc-23.6-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/psmisc/23.6_i686/psmisc-23.6-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/psmisc/23.6_x86_64/psmisc-23.6-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: 'd951db8224299af61b2ca8fb78d64f93667f52df0901928927d8c743216a629f',
-     armv7l: 'd951db8224299af61b2ca8fb78d64f93667f52df0901928927d8c743216a629f',
-       i686: '00181538fdcdb8922a07ff68f046a1d5333866f76bb241ae628d369179a931c2',
-     x86_64: '61880a1987c8295f18c44c8472a953179510318082ed92ac1ade306dbda96bd1'
+    aarch64: 'd84be50b9c1f15bbd6e547e3b5b323e3218c9e02cd348792afce0106108e09de',
+     armv7l: 'd84be50b9c1f15bbd6e547e3b5b323e3218c9e02cd348792afce0106108e09de',
+       i686: 'f833aae432564a90811d6196b0353ddeca6a3c818532d05fd161731deb3d960d',
+     x86_64: 'c11276bfd53756012bf7375dcf4a7add56799d29456ad78a9ec0442bfddc9365'
   })
 
+  depends_on 'gcc' # R
+  depends_on 'glibc' # R
+  depends_on 'ncurses' # R
+
+  def self.patch
+    system "sed -i 's,<curses.h,<ncurses/curses.h,g' src/pstree.c"
+    system "sed -i 's,<term.h,<ncurses/term.h,g' src/pstree.c"
```

Comment:
I've experienced this issue on several occasions.  I think this can also be solved with `CFLAGS+=" -I#{CREW_PREFIX}/include/ncurses"`.

Reply:
Done